spec/0.0.2/provider.json

```diff
@@ -14,6 +14,41 @@
             "type": "string",
             "value": "null"
         },
+        "repository": {
+            "type": "object",
+            "properties": {
+                "source": {
+                    "name": "source",
+                    "description": "Source location of the source control repository. The source MUST be specified by a valid URL.",
+                    "type": "string",
+                    "value": "null"
```

Comment:
This should say: `"default": "null"` (just like all other lines containing the `"value"` key). I forgot to replace some occurrences of `"value"` instead of `"default"` in the spec by mistake. I'll issue a separate bug and PR to fix that everywhere.